adapters/handlers/grpc/server.go

```diff
@@ -312,7 +324,30 @@ func extractPropertiesAnswer(results map[string]interface{}, properties search.S
 	return &props, nil
 }
 
-func extractPropertiesRequest(reqProps *pb.Properties) []search.SelectProperty {
+func getAllNonRefNonBlobProperties(scheme schema.Schema, className string) ([]search.SelectProperty, error) {
```

Comment:
A comment explaining why we have to get all non ref/blob props separately would be helpful

Reply:
There is a comment at the place where this is called :) 
```
		// no return values selected, return all properties and metadata. Ignore blobs and refs to not overload the
		// response

```